learning-objectives/intl/es.yml

```diff
@@ -606,6 +606,208 @@ auth/jwt:
 auth/password-access-and-storage:
   title: Almacenamiento y acceso de contraseñas
 
+java:
+  title: Java
+  links:
+    - title: Example title
+      url: https://example.com
+
+java/access-modifiers:
+  title: Modificadores de acesso (public, private, protected)
+  
+java/variables:
+  title: Variables
+
+java/conditionals:
+  title: Uso de condicionales
+  
+java/loops:
+  title: Uso de bucles (Loops)
+
+java/data-types:
+  title: Tipos de data
+
+java/data-types/primitive:
+  title: Primitivos
+
+java/data-types/primitive-vs-non-primitive:
+  title: Datos primitivos vs no primitivos
+    
+java/data-stypes/strings:
+  title: Cadenas
+
+java/data-types/arrays:
+  title: Arreglos
+  
+java/collections:
+  title: Colecciones
+
+java/collections/array-list:
+  title: ArrayList
+  
+java/collections/hash-map:
+  title: HashMap
+
+java/collections/hash-set:
+  title: HashSet
+
+java/testing:
+  title: Testing
+
+java/testing/junit:
+  title: JUnit
+
+java/testing/mockito:
+  title: Mockito
+
+java/spring-framework:
+  title: Spring Framework
+
+java/spring-framework/initializr:
+  title: Initializr
+  links:
+    - title: Core Technologies
+      url: https://docs.spring.io/spring-framework/reference/core.html
```

Comment:
@cros410 @NaraSakamoto 
hay algo se llama "Core Concepts" en los readmes y es dificil saber si Core Concepts engloba initializr, beans, inversion of control, y annotations - o es parte de initializer solo
https://github.com/Laboratoria/UPSK-JAVA-001-fleet-management-api/blob/main/README.md#conceptos-core

Reply:
Cambia a lo que hablamos en el huddle y agregue el link, pero me avisas si mejor mover initializr abajo de core concepts